openedx/core/djangoapps/team_api/utils.py

```diff
@@ -0,0 +1,18 @@
+import unicodedata
+import re
+
+from django.utils.encoding import force_unicode
+from django.utils.safestring import mark_safe
+
+
+# Taken from Django 1.8 source code because it's not supported in 1.4
+def slugify(value):
+    """
+    Converts to ASCII. Converts spaces to hyphens. Removes characters that
+    aren't alphanumerics, underscores, or hyphens. Converts to lowercase.
+    Also strips leading and trailing whitespace.
+    """
+    value = force_unicode(value)
+    value = unicodedata.normalize('NFKD', value).encode('ascii', 'ignore').decode('ascii')
+    value = re.sub('[^\w\s-]', '', value).strip().lower()
```

Comment:
Pylint is worried about this regex -- it should be a raw string to avoid that warning.